.github/workflows/main-merge.yml

```diff
@@ -0,0 +1,18 @@
+# Merges any changes from release/prerelease to main (e.g. servicing changes)
+
+name: Flow main to release/dev18.0
+on:
+  schedule:
+    # once a day at 13:00 UTC to cleanup old runs
+    - cron: '0 13 * * *'
+  workflow_dispatch:
```

Comment:
Workflow dispatch jobs allow you to choose the branch to run against. In this case we would want to run against main. https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#workflow_dispatch